public/app/core/components/TransformersUI/SortByTransformerEditor.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useMemo } from 'react';
+import { DataTransformerID, standardTransformers, TransformerRegistyItem, TransformerUIProps } from '@grafana/data';
+import { getAllFieldNamesFromDataFrames } from './OrganizeFieldsTransformerEditor';
+import { InlineField, InlineFieldRow, Select, Switch } from '@grafana/ui';
+
+import { SortByField, SortByTransformerOptions } from '@grafana/data/src/transformations/transformers/sortBy';
+
+export const SortByTransformerEditor: React.FC<TransformerUIProps<SortByTransformerOptions>> = ({
+  input,
+  options,
+  onChange,
+}) => {
+  const fieldNames = useMemo(
+    () =>
+      getAllFieldNamesFromDataFrames(input).map(n => ({
+        value: n,
+        label: n,
+      })),
+    [input]
+  );
+
+  // Only supports single sort for now
+  const onSortChange = useCallback(
+    (idx: number, cfg: SortByField) => {
+      onChange({ ...options, sort: [cfg] });
+    },
+    [options]
+  );
+
+  const sorts = options.sort?.length ? options.sort : [{} as SortByField];
+
+  return (
+    <div>
+      {sorts.map((s, index) => {
+        return (
+          <InlineFieldRow key={`${s.field}/${index}`}>
+            <InlineField label="Field" labelWidth={10} grow={true}>
+              <Select
+                options={fieldNames}
+                value={fieldNames.find(v => v.value === s.field)}
+                placeholder="Select field"
+                onChange={v => {
+                  onSortChange(index, { ...s, field: v.value! });
+                }}
+              />
+            </InlineField>
+            <InlineField
+              label="Reverse"
+              onClick={() => {
+                onSortChange(index, { ...s, desc: !!!s.desc });
+              }}
+            >
+              <Switch
```

Comment:
@dprokop is there a way to get this centered reasonably without custom emotion css?

Reply:
There is an InlineSwitch